cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -1401,16 +1429,30 @@ void make_json_column(json_column& root_column,
   root_column.level_child_cols_recursively(root_column.current_offset);
 }
 
+auto casting_options(cudf::io::json_reader_options const& options)
```

Comment:
Dumb question, but why is a function returning `parse_options` called `casting_options`?
The returned value is seemingly used in type inference and parsing.

Reply:
Fixed naming 👍